tests/qgis-projects/tests/replace_database_connexion_qgs_project.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+DIR="$1"
+
+echo "$DIR : Is the path for the qgs projects to modify"
+
+echo "#### Update files ####"
+
+for file in "$DIR"*.qgs
+do
+    echo "$file updated"
+    sed -i "s/source=\"service='lizmapdb'/source=\"dbname='lizmap_demo_lizmap_3_0' host=qgisdb-snap.lizmap.com port=5432 user='$2'/g" $file;
```

Comment:
`lizmap_demo_lizmap_3_0`, I guess we want to use it as well for lizmap 3.1, 3.2... 3.5, so why hard coding 3.0 ?

Reply:
I check that